harvard-university-of-worcester.csl

```diff
@@ -261,14 +261,14 @@
       </else>
     </choose>
   </macro>
-  <citation collapse="year" disambiguate-add-year-suffix="true" et-al-use-first="3" et-al-min="4">
+  <citation et-al-min="3" et-al-use-first="1" sort-separator="," disambiguate-add-year-suffix="true" collapse="year" cite-group-delimiter=" ">
```

Comment:
@University-of-Worcester --so as to not confuse you: while there is currently a problem with how that attribute works, for your purposes you'll definitely just want to delete it. In your case it won't make a difference in Zotero and likely Mendeley for now, but it might in the future.